starboard/android/shared/get_home_directory.cc

```diff
@@ -20,24 +20,19 @@
 #include "starboard/android/shared/file_internal.h"
 #include "starboard/common/log.h"
 #include "starboard/common/string.h"
-#include "starboard/shared/nouser/user_internal.h"
+#include "starboard/shared/starboard/get_home_directory.h"
 
 using ::starboard::android::shared::g_app_files_dir;
 
 namespace starboard {
 namespace shared {
-namespace nouser {
+namespace starboard {
```

Comment:
This seems weirdly out of place and a bit .. circular, but it matches the header, so makes sense. No action needed